src/js/utils/colors.js

```diff
@@ -1,7 +1,38 @@
+// Track which deprecated colors have already been warned about
+const warnedColors = new Set();
+
 // Returns the specific color that should be used according to the theme.
 // If 'dark' is supplied, it takes precedence over 'theme.dark'.
 // Can return undefined.
 export const normalizeColor = (color, theme, dark) => {
+  if (
```

Comment:
Would be nice to pull this chunk into a separate (local) function so normalizeColor is more readable. This kinda dominates now